docs/interpreter/cassandra.md

```diff
@@ -185,74 +193,96 @@ Insert into users(login,name) vAlues('hsue','Helen SUE');
 ```
 
 ## Syntax Validation
-The interpreters is shipped with a built-in syntax validator. This validator only checks for basic syntax errors.
-All CQL-related syntax validation is delegated directly to **Cassandra**.
 
-Most of the time, syntax errors are due to **missing semi-colons** between statements or **typo errors**.
+The interpreters is shipped with a built-in syntax validator. This validator only checks for basic syntax errors. 
+
+All CQL-related syntax validation is delegated directly to **Cassandra** 
 
+Most of the time, syntax errors are due to **missing semi-colons** between statements or **typo errors**.
+                                    
 ## Schema commands
```

Comment:
After applying this PR, I found some uselessly generated `<br>` tag here. 
![screen shot 2016-02-05 at 11 06 18 am](https://cloud.githubusercontent.com/assets/10060731/12836076/d8c6118a-cbf8-11e5-8e20-9b33008583ca.png)
Could you checkout this please ?  : )